drivers/sensor/apds9960/apds9960.h

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _SENSOR_APDS9960_H_
+#define _SENSOR_APDS9960_H_
+
+#include <device.h>
```

Comment:
agree, printk is not being used in your code, so not sure why you are adding it here. The headers need to be included in the source file where they are using and and not via another header.